spec/views/generic_file/edit.html.erb_spec.rb

```diff
@@ -40,4 +40,34 @@
       expect(page).to have_selector("select#generic_file_resource_type", count: 1)
     end
   end
+
+  describe 'form elements' do
+    let(:generic_file) do
+      stub_model(GenericFile, id: '123',
+                              depositor: 'bob',
+                              resource_type: ['Book', 'Dataset'])
+    end
+
+    let(:form) do
+      Sufia::Forms::GenericFileEditForm.new(generic_file)
+    end
+
+    before do
+      allow(controller).to receive(:current_user).and_return(stub_model(User))
+      assign(:generic_file, generic_file)
+      assign(:form, form)
+      assign(:version_list, [])
+    end
+
+    let(:page) do
+      render
+      Capybara::Node::Simple.new(rendered)
+    end
+
+    it "onlies draw one resource_type multiselect" do
+      within('#descriptions_display') do
+        expect(page).to have_button('Save Descriptions')
+      end
+    end
+  end
```

Comment:
you shouldn't have to replicate the entire setup for the test. The above section does it for you. Can you just move lines 67-71 to below 41?